transport/grpc/outbound.go

```diff
@@ -126,8 +126,11 @@ func (o *Outbound) Call(ctx context.Context, request *transport.Request) (*trans
 	if err != nil {
 		return nil, err
 	}
+	buf := bufferpool.NewAutoReleaseBuffer()
+	buf.Write(responseBody)
```

Comment:
ResponseBody slice is allocated and holds the response data. Creating new buffer costs an additional allocation plus cost of copying the buffer. Also, encoding layer holds the reference to the body until the response object is marked by gc

Reply:
You're right. The problem I'm trying to fix doesn't exist in the grpc implementation.